ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/condition/ConditionNodeState.java

```diff
@@ -3,16 +3,12 @@
 import behavior_msgs.msg.dds.ConditionNodeStateMessage;
 import us.ihmc.behaviors.behaviorTree.BehaviorTreeRootNodeState;
 import us.ihmc.behaviors.behaviorTree.LeafNodeState;
-import us.ihmc.communication.crdt.CRDTStatusBoolean;
-import us.ihmc.communication.ros2.ROS2ActorDesignation;
 
 public class ConditionNodeState extends LeafNodeState<ConditionNodeDefinition>
 {
    private final CounterConditionState counter;
    private final LLMConditionState llm;
    private final ProximityConditionState proximityCheck;
-   private final CRDTStatusBoolean conditionMet;
-   private final CRDTStatusBoolean evaluatingCondition;
```

Comment:
Remove Luigi's hacks from when I broke my knee and couldn't review it :laughing: